src/css/styles.scss

```diff
@@ -1,46 +1,26 @@
-// Load CSS Reset from NPM
-// CSS resets "normalize" how browser styles are applied.
-// Try uncommenting this and saving. What changes in the browser?
-@import "~normalize.css";
+body {background-color: #F8F9EB;
+	margin-left: 5em;
+	margin-right: 5em;
+	margin-bottom: 5em;}
```

Comment:
I have a feeling this margin might be a bit large on mobile - be sure to check it!